src/components/annotations/AnnotationPage.js

```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import { connect } from "react-redux";
 import { useParams } from "react-router-dom";
-import { fetchAnnotationDetails } from "../../actions";
```

Comment:
Move all the annotation files into 'annotations' folder.
Just updated the import path.